extensions/amp-ad/0.1/amp-ad-xorigin-iframe-handler.js

```diff
@@ -523,6 +529,17 @@ export class AmpAdXOriginIframeHandler {
       this.intersectionObserver_.fire();
     }
   }
+
+  /**
+   * @param {!string} message
+   * @private
+   */
+  userErrorForAnalytics_(message) {
+    const e = new Error();
```

Comment:
Can we check message is a string here. Just want to be safe.
Then do `e = new Error(message);`